docs/zh-CN/sql-reference/sql-statements/Data Manipulation/SHOW TABLET STORAGE FORMAT.md

```diff
@@ -0,0 +1,53 @@
+---
+{
+    "title": "SHOW TABLET STORAGE FORMAT",
+    "language": "zh-CN"
+}
+---
+
+<!-- 
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# SHOW TABLET STORAGE FORMAT
+## description
+    该语句用于显示Backend上的存储格式信息（仅管理员使用）
+    语法：
+        SHOW TABLET STORAGE FORMAT [VERBOSE]
+
+## example
+    MySQL [(none)]> show tablet storage format;
```

Comment:
Maybe "show backends with storage format" is better? because I find the result is organized by backend, not tablet.